### DISCUSSION TITLE
OpenAI Function Token Count

### DISCUSSION THREAD

**Discussion by CeGyger92** [microsoft/Tokenizer#24] [Q&A]
Is there support for counting tokens from function definitions? How would one take a function definition from the Azure.AI.OpenAI package below and count the tokens?

List<FunctionDefinition> functions = new List<FunctionDefinition>()
                        {
                            new FunctionDefinition()
                            {
                                Name = "Function",
                                Description = "Description",
                                Parameters = BinaryData.FromString("{\"type\": \"object\",\"properties\": {},\"require\": []}")
                            },
                        };

**Comment by shengyfu** [comment]
It would be the same as any other string.
